aiida_registry/fetch_metadata.py

```diff
@@ -65,11 +100,19 @@ def complete_plugin_data(
     REPORTER.info(f'{plugin_data["package_name"]}')
 
     plugin_data["hosted_on"] = get_hosted_on(plugin_data["code_home"])
+
+    if plugin_data["hosted_on"] == "github.com":
```

Comment:
this should also be conditioned on whether a token was actually provided.

plus, the entire if/else block here can fail if, for some reason, the plugin fails to be cloned - let's put a try/except around it